otcdocstheme/theme/otcdocs/header.html

```diff
@@ -298,6 +298,7 @@
               <span class="padding-slot-heading" slot="heading">Other</span>
               <ul>
                 <li><a href="{{base_url}}/enterprise-dashboard/index.html">Enterprise Dashboard</a></li>
+                <li><a href="{{base_url}}/price-calculator/index.html" target="_blank" rel="noopener noreferrer">Price API</a></li>
```

Comment:
the name sounds strange. This should be the name from the stakeholders, not something like 'Price Calculator API'